CMakeLists.txt

```diff
@@ -53,20 +53,30 @@ set(RAJA_VERSION_PATCHLEVEL 0)
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake/thirdparty" ${CMAKE_MODULE_PATH})
 
 # Build options
-option(RAJA_ENABLE_OPENMP "Build OpenMP support" On)
-option(RAJA_ENABLE_TARGET_OPENMP "Build OpenMP on target device support" Off)
-option(RAJA_ENABLE_CUDA "Build CUDA support" Off)
-option(RAJA_ENABLE_CLANG_CUDA "Use Clang's native CUDA support" Off)
-set(RAJA_CUDA_ARCH "sm_35" CACHE STRING "Compute architecture to pass to CUDA builds")
-option(RAJA_ENABLE_CUB "Use cub for scans using CUDA" On)
-option(RAJA_ENABLE_TESTS "Build tests" On)
-option(RAJA_ENABLE_EXAMPLES "Build simple examples" On)
-option(RAJA_ENABLE_NESTED "Enable nested loop support" Off)
-option(RAJA_ENABLE_WARNINGS "Enable warnings as errors for CI" Off)
-option(RAJA_ENABLE_DOCUMENTATION "Build RAJA documentation" Off)
+set(ENABLE_OPENMP On CACHE Bool "Build OpenMP support")
```

Comment:
is there a reason why this is changed from option to set? (same for CUDA and COPY_HEADERS)

Also, what does COPY_HEADERS do? (it's undocumented)